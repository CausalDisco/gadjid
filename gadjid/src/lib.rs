// SPDX-License-Identifier: MPL-2.0
#![warn(missing_docs)]
//! gadjid -  Graph Adjustment Identification Distance library
mod ascending_list_utils;
mod graph_loading;
pub mod graph_operations;
mod partially_directed_acyclic_graph;

pub use graph_loading::constructor::EdgelistIterator;
pub use partially_directed_acyclic_graph::LoadError;
pub use partially_directed_acyclic_graph::PDAG;
