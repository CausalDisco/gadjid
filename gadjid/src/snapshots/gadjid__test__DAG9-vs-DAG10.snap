---
source: src/lib.rs
expression: "test(\"10009.DAG-100\", \"10010.DAG-100\")"
---
g_guess: 10010.DAG-100
g_true: 10009.DAG-100
t: 34
y: 18
z:
  - 0
  - 2
  - 4
  - 7
  - 8
  - 9
  - 11
  - 12
  - 13
  - 14
  - 16
  - 17
  - 18
  - 19
  - 21
  - 26
  - 29
  - 30
  - 31
  - 32
pa_true_T:
  - 0
  - 26
  - 23
  - 25
  - 19
  - 16
  - 13
  - 4
  - 1
  - 30
  - 33
  - 24
  - 15
  - 12
  - 6
an_true_T:
  - 0
  - 34
  - 28
  - 25
  - 22
  - 19
  - 16
  - 13
  - 10
  - 7
  - 4
  - 1
  - 32
  - 29
  - 26
  - 23
  - 20
  - 17
  - 14
  - 11
  - 8
  - 5
  - 2
  - 33
  - 30
  - 27
  - 24
  - 21
  - 18
  - 15
  - 12
  - 9
  - 6
  - 3
ch_true_T:
  - 73
  - 88
  - 58
  - 93
  - 84
  - 49
  - 43
  - 72
  - 37
  - 69
  - 75
  - 60
  - 95
  - 45
  - 56
  - 53
  - 82
  - 47
  - 85
  - 41
  - 70
  - 67
de_true_T:
  - 61
  - 55
  - 49
  - 43
  - 37
  - 98
  - 92
  - 86
  - 80
  - 74
  - 68
  - 62
  - 56
  - 50
  - 44
  - 38
  - 99
  - 93
  - 87
  - 81
  - 75
  - 69
  - 63
  - 57
  - 51
  - 45
  - 94
  - 88
  - 82
  - 76
  - 70
  - 64
  - 58
  - 52
  - 46
  - 40
  - 34
  - 95
  - 89
  - 83
  - 77
  - 71
  - 65
  - 59
  - 53
  - 47
  - 41
  - 96
  - 90
  - 84
  - 78
  - 72
  - 66
  - 60
  - 54
  - 48
  - 42
  - 97
  - 91
  - 85
  - 79
  - 73
  - 67
poss_de_true_T:
  - 61
  - 55
  - 49
  - 43
  - 37
  - 98
  - 92
  - 86
  - 80
  - 74
  - 68
  - 62
  - 56
  - 50
  - 44
  - 38
  - 99
  - 93
  - 87
  - 81
  - 75
  - 69
  - 63
  - 57
  - 51
  - 45
  - 94
  - 88
  - 82
  - 76
  - 70
  - 64
  - 58
  - 52
  - 46
  - 40
  - 34
  - 95
  - 89
  - 83
  - 77
  - 71
  - 65
  - 59
  - 53
  - 47
  - 41
  - 96
  - 90
  - 84
  - 78
  - 72
  - 66
  - 60
  - 54
  - 48
  - 42
  - 97
  - 91
  - 85
  - 79
  - 73
  - 67
prop_an_true_Y:
  - 0
  - 17
  - 14
  - 11
  - 8
  - 5
  - 2
  - 13
  - 10
  - 7
  - 4
  - 1
  - 18
  - 12
  - 9
  - 6
  - 3
nam: []
nva:
  - 0
  - 61
  - 55
  - 49
  - 43
  - 37
  - 98
  - 31
  - 92
  - 25
  - 86
  - 19
  - 80
  - 13
  - 74
  - 7
  - 68
  - 1
  - 62
  - 56
  - 50
  - 44
  - 38
  - 99
  - 32
  - 93
  - 26
  - 87
  - 20
  - 81
  - 14
  - 75
  - 8
  - 69
  - 2
  - 63
  - 57
  - 51
  - 45
  - 39
  - 33
  - 94
  - 27
  - 88
  - 21
  - 82
  - 15
  - 76
  - 9
  - 70
  - 3
  - 64
  - 58
  - 52
  - 46
  - 40
  - 95
  - 28
  - 89
  - 22
  - 83
  - 16
  - 77
  - 10
  - 71
  - 4
  - 65
  - 59
  - 53
  - 47
  - 41
  - 35
  - 96
  - 29
  - 90
  - 23
  - 84
  - 17
  - 78
  - 11
  - 72
  - 5
  - 66
  - 60
  - 54
  - 48
  - 42
  - 36
  - 97
  - 30
  - 91
  - 24
  - 85
  - 18
  - 79
  - 12
  - 73
  - 6
  - 67
shd:
  - 0.4951515151515152
  - 2451
sid:
  - 0.7182828282828283
  - 7111
oset_aid:
  - 0.3409090909090909
  - 3375
parent_aid:
  - 0.7182828282828283
  - 7111
ancestor_aid:
  - 0.26171717171717174
  - 2591
